packages/consent/consent-tools/src/types/settings.ts

```diff
@@ -56,9 +54,7 @@ export interface CreateWrapperSettings {
    * },
    * ```
    **/
-  shouldLoadSegment?: (
-    ctx: LoadContext
-  ) => Categories | void | Promise<Categories | void>
+  shouldLoadSegment?: (ctx: LoadContext) => void | Promise<void>
```

Comment:
This looks like a breaking change - are we doing a major version bump (totally cool with it if we are!)

Reply:
Yes, was going to do a major bump!